static/css/datatable.css

```diff
@@ -0,0 +1,13 @@
+body{
```

Comment:
Šo te nevajag, es iznesu ārā partials mūsu galveno html failu ar visiem css un js importiem, tāpēc šis atkārtojas.